src/core/methods/createNeighbourhoodMeta.ts

```diff
@@ -22,6 +23,14 @@ export async function createNeighbourhoodMeta(
     })
   );
 
+  expressionLinks.push(
+    new Link({
+      source: "self",
+      target: creatorDid,
+      predicate: "rdf://creator",
```

Comment:
How about making the plain text (`self` and `rdf://creator` here) the constants in a registry of such terms and import them when needed?

Reply:
Good idea. I think it would be a good idea to do the same with the profile data as well. Some places we reference fields like this `user['foaf:AccountName']`. Would be nice to make this somewhat typesafe by creating enums/constants and maybe a helper function for converting profile data to map back to the `Profile` type in `store/types.ts`.